### PR TITLE
Added sample playbooks for AKS

### PR DESCRIPTION
**Pull request by caohai**
## Purpose

* Added sample playbooks for azure_rm_aks module
  - aks_create.yml: create an AKS cluster
  - aks_scale.yml: scale an existed AKS cluster by changing node count

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## How to Test
*  Install ansible

```bash
$ pip install ansible[azure]
```

*  [Optional] Install ansible role

```bash
$ ansible-galaxy install Azure.azure_preview_modules
$ pip install -r ~/.ansible/roles/Azure.azure_preview_modules/files/requirements-azure.txt
```

*  Get the sample playbook

```
git clone https://github.com/Azure-Samples/ansible-playbooks.git
cd ansible-playbooks
git checkout [branch-name]
```

* Test the code

```bash
cd azure_rm_aks
ansible-playbook aks_create.yml
ansible-playbook aks_scale.yml
```

## What to Check
Verify that the playbook is successfully run.

### CODE REVIEW

File: azure_rm_aks/aks_create.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- name: Create Azure Kubernetes Service
+  hosts: localhost
+  connection: local
+  vars:
+    resource_group: "{{ resource_group_name }}"
+    location: eastus
+    aks_name: myAKSCluster
+    username: azureuser
+    ssh_key: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDH14vPP1NE+qc3SgpoSPYMVaPLj8ouA1azujnMPbVKtN8mSg0KBWVWebx+GLqGI9nrKwmVwPHVzM8LhbbpYhfnN0UA2OV7sxBIshy7BshDTewiLyR9FXuNjuf8Cwd7WI+vgkKO0Yh6EdgwZ2hz+yVTn+5umCRDOg0ht0i2kyPMn1MHv61aN3noYp5zV8W3Tg8oJ2DPei6FxcNheRWkyvjan73/rISQJYWURlI90b3MCQOTXuI2w6BIxF3kHIHLUtzD7FqQhZdEkM/8cA9wUDgnS6+3ySkGOuQazkEWYn60Ll/597u2fLsq6P/pOCM1+KWLf2kY6mTl/dNxiIdV1d01
+    client_id: "{{ your_client_id }}"
```
Comment (yuwzho) [inline azure_rm_aks/aks_create.yml]: "{{ lookup('env', 'AZURE_CLIENT_ID') }}"

We need to make sure all the sample can be run on the integration test, so here retrieve the credentials from env.

Comment (caohai) [reply azure_rm_aks/aks_create.yml]: OK, I'll change them.

**Comment by kyliel** [comment]
How do you get resource group name? And don't list the complete SSH key in your sample.

**Comment by caohai** [comment]
@kyliel 
- For resource group name, I'm following the convention of most sample playbooks in this repo.[Examples: [vm_create_ssh](https://github.com/Azure-Samples/ansible-playbooks/blob/master/vm_create_ssh.yml)] I'm not sure how resource group name is retrieved, @yuwzho can you give some explaination?
- I checked the SSH key in both playbooks and I believe it's complete.

**Comment by kyliel** [comment]
vm_create_ssh should be fixed too.

**Comment by yuwzho** [comment]
@kyliel @caohai The resource group is passed to playbook dynamically when running the test. 

And the SSH, I don't think this is a problem writing in the playbook, since the public key can only be used paired with secret one.

**Comment by yuwzho** [comment]
@caohai I m not sure whether the invalid SSH can successfully create an AKS, could you please have a try? Otherwise, you can store the key in env, and retrieve it in the playbook

**Pull request merged**